src/main/java/spring/intro/config/WebConfig.java

```diff
@@ -0,0 +1,23 @@
+package spring.intro.config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.servlet.config.annotation.EnableWebMvc;
+import org.springframework.web.servlet.view.InternalResourceViewResolver;
+import org.springframework.web.servlet.view.JstlView;
+
+@Configuration
+@ComponentScan(basePackages = "spring.intro.controllers")
+@EnableWebMvc
+public class WebConfig {
+
+    @Bean
+    public InternalResourceViewResolver resolver() {
```

Comment:
```suggestion
    public InternalResourceViewResolver createViewResolver()  {
```
method's names should start with verb

Reply:
хорошо